tomster-player/app/templates/components/album-list.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="list-group sidebar">
+<div class="list-group {{if show-songs '' 'sidebar'}}">
```

Comment:
done, this was referring to `album-tile` not `album-list` 👍  https://github.com/simplabs/ember-workshop/commit/ff6daa68e5e28cd63c9a160fb59062173173b96e